modules/dag-l1/src/main/scala/org/tessellation/dag/l1/domain/trust/storage/TrustStorage.scala

```diff
@@ -0,0 +1,12 @@
+package org.tessellation.dag.l1.domain.trust.storage
+
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.schema.trust._
+
+trait TrustStorage[F[_]] {
+  def getTrustScore(peerId: PeerId): F[Option[Score]]
```

Comment:
Just `getScore`.

For example in `AddressStorage` we have `getBalance` instead of `getAddressBalance` so you should name it `getScore`

Reply:
Changed.